### PR TITLE
update SetStepFunctionWrapper.sol

### PR DESCRIPTION
**Pull request by ilanDoron**
add it to compileContracts.js

### CODE REVIEW

File: contracts/wrappers/SetStepFunctionWrapper.sol [hunk 1]
```diff
@@ -25,22 +26,32 @@ contract SetStepFunctionWrapper is Withdrawable {
     }
 
     function setImbalanceStepFunction(ERC20 token,
```
Comment (Anyhowclick) [inline contracts/wrappers/SetStepFunctionWrapper.sol]: shift the `ERC20 token` to the next line?

File: contracts/wrappers/SetStepFunctionWrapper.sol [hunk 2]
```diff
@@ -25,22 +26,32 @@ contract SetStepFunctionWrapper is Withdrawable {
...
+        int[] xBuy,
+        int[] yBuy,
+        int[] xSell,
+        int[] ySell)
+    public onlyOperator
+    {
         uint i;
 
-        // check all x for buy are positive and y are negative
+        // check all x for buy are positiv
```
Comment (Anyhowclick) [inline contracts/wrappers/SetStepFunctionWrapper.sol]: positive lol.

File: contracts/wrappers/SetStepFunctionWrapper.sol (hunk 1 above)
Comment (ilanDoron) [reply contracts/wrappers/SetStepFunctionWrapper.sol]: done

**Comment by ilanDoron** [comment]
will be added in another PR

**Pull request closed without merging**